docs/ops/arithmetic/Erf_1.md

```diff
@@ -13,17 +13,15 @@ For each element from the input tensor calculates corresponding element in the o
 erf(x) = \pi^{-1} \int_{-x}^{x} e^{-t^2} dt
 \f]
 
-**Attributes**:
-
-    No attributes available.
+**Attributes**: *Erf* operation has no attributes.
 
 **Inputs**
 
-* **1**: A tensor of type *T*. **Required.**
+* **1**: A tensor `x` of type *T* and arbitrary shape. **Required.**
```

Comment:
nitpick: In general when there is only one input we do not label it with a name (for simplicity). We refer to it as input tensor. You can apply this to Output section too